### PR TITLE
Polish translation

### PR DESCRIPTION
**Pull request by rtpm**

### CODE REVIEW

File: lektor/translations/pl.json [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+  "CREATE_CHILD_PAGE": "Dodaj podstronę",
+  "DELETE_ATTACHMENT_PROMPT": "Czy na pewno chcesz usunąć ten załącznik?",
+  "DELETE_ATTACHMENT_ALT_PROMPT": "Czy na pewno chcesz usunąć dane załącznika dla tego języka?",
+  "DELETE_PAGE_PROMPT": "Czy na pewno chcesz usunąć tą stronę?",
+  "DELETE_PAGE_ALT_PROMPT": "Czy na pewno chcesz usunąć ten język?",
+  "DELETE_PAGE_CHILDREN_WARNING": "To spowoduje także usunięcie wszystkich podstron.",
+  "DELETE_RECORD": "Skasuj “%s”",
+  "DELETE_ALL_PAGE_ALTS": "Usuń również wszystkie wersje językowe oraz załączniki.",
+  "DELETE_ALL_ATTACHMENT_ALTS": "Usuń wszystkie wersje językowe oraz załączony plik.",
+  "DELETE_ONLY_PRIMARY_PAGE_ALT": "Usuń tylko podstawowy rekord.  Załączniki, wersje językowe oraz podstrony nie będę usunięte.",
```
Comment (krzysztof-sikorski) [inline lektor/translations/pl.json:47]: Extra space after first sentence - probably unintentional.

**Comment by rtpm** [comment]
Ahh.. Yes, I made some changes before the commit. 

Feel free to modify the translation as some words were not easy to translate due to multiple meaning.

Thanks!

**Comment by krzysztof-sikorski** [comment]
The translation is good, I just have a bit of OCD concerning white-space so I nitpicked :)

**Comment by singingwolfboy** [comment]
I don't speak Polish, but I'm going to assume that this translation is correct, and merge it. Thank you! ✨

**Pull request merged**